cuda/runtime/BUILD.local_cuda

```diff
@@ -298,3 +298,158 @@ cc_library(
         ":nvml_lib",
     ]),
 )
+
+# curand
+cc_import(
+    name = "curand_so",
+    shared_library = "cuda/lib64/libcurand.so",
+    target_compatible_with = ["@platforms//os:linux"],
+)
+
+cc_import(
+    name = "curand_lib",
+    interface_library = "cuda/lib/x64/curand.lib",
+    system_provided = 1,
+    target_compatible_with = ["@platforms//os:windows"],
+)
+
+cc_library(
+    name = "curand",
+    deps = [
+        ":cuda_headers",
+    ] + if_linux([
+        ":curand_so",
+    ]) + if_windows([
+        ":curand_lib",
+    ]),
+)
+
+# nvcc
+cc_library(
+    name = "nvcc",
```

Comment:
Why is this named as `nvcc`. I think nvptx will be more reasonable. It also needs nvptxcompiler_static.lib or libnvptxcompiler_static.a to work properly

Reply:
good catch, thanks. updated.